docs/releases/5.0.txt

```diff
@@ -129,6 +129,28 @@ sets a database-computed default value. For example::
         created = models.DateTimeField(db_default=Now())
         circumference = models.FloatField(db_default=2 * Pi())
 
+Choices defined as mappings
+---------------------------
+
+Existing :attr:`.Field.choices` and ChoiceField's
+:attr:`~django.forms.ChoiceField.choices` now accept mappings as well as
+iterables, and also accept :ref:`Choices classes <field-choices-enum-types>`
+directly instead of requiring expansion with the ``choices`` attribute. For
+example::
+
+    MEDIA_CHOICES = {
+        "Audio": {
+            "vinyl": "Vinyl",
+            "cd": "CD",
+        },
+        "Video": {
+            "vhs": "VHS Tape",
+            "dvd": "DVD",
+        },
+        "unknown": "Unknown",
+    }
```

Comment:
This doesn't really match the description of new features: _"...now accept mappings .. also accept Choices classes directly ... "_. I'd add an example model.

Reply:
Personally, I'd not mix descriptions of these two features.

Reply:
Will tweak.